src/sesclient.native/README.md

```diff
@@ -5,14 +5,17 @@ This command line executable demonstrates use of the [native-code software entit
 **This is draft documentation subject to change.**
 
 ## Working with sesclient.native
+
 This executable is intended to be used in conjunction with the [sestest command line utility](../sestest) to generate tokens.
 
 The executable expects the following parameters (in any order):
 
-| Parameter            | Required  | Definition                                                                                                                         |
-| -------------------- | --------- | ---------------------------------------------------------------------------------------------------------------------------------- |
-| --url                | Mandatory | The URL at which the server should process requests. <br/> **Note**: must start with `https:`.                                      |
-| --thumbprint         | Mandatory | Thumbprint of a certificatein the server's SSL certificate chain. <br/> **Note**: cannot be the thumbprint of the root certificate. |
-| --common-name        | Mandatory | The common name of the certificate indicated by `--thumbprint`.                                                                    |
-| --token              | Mandatory | The token as returned by `sestest generate`.                                                                                       |
-| --application        | Mandatory | Unique identifier for the application being requested.                                                                             |
+|   Parameter   | Required  |                                                                   Definition                                                                    |
+| ------------- | --------- | ----------------------------------------------------------------------------------------------------------------------------------------------- |
+| --url         | Mandatory | The URL of the server that will process our request. <br/> **Note**: must start with `https:`.                                                  |
+| --token       | Mandatory | The token as returned by `sestest generate`.                                                                                                    |
+| --application | Mandatory | Unique identifier for the application being requested.                                                                                          |
+| --thumbprint  | Optional  | Thumbprint of an additional certificate to accept in the TLS certificate chain. <br/> **Note**: cannot be the thumbprint of a root certificate. |
```

Comment:
`--thumbprint` is for if I am using `sestest server` for verification, and I need to set `--thumbprint` to be the TLS certificate I specified for that?  Is this explained in the workflow/how-to documentation?

Reply:
Oh, or this refers to a token signing cert, not to HTTPS negotiation?  Bit confused by discussion in paragraph below...

Reply:
Both `--thumbprint` and `--common-name` are used to specify an additional certificate that can be used (when testing) for the HTTPS connection. I've reworded the parameter descriptions and completely rewritten the paragraph below.